src/components/d2l-activity-question-points.js

```diff
@@ -126,16 +129,16 @@ class ActivityQuestionPoints extends BaseMixin(LitElement) {
 				</div>
 			</d2l-list-item-content>
 			<div class="activity_list__points_input" slot="actions">
-				<label for="points_input_${question.id}" class="points_input__label">
+				<label for="points_input_${question.id}" class="points_input__label d2l-label-text">
 					${this.localize('inputLabelPoints')}
 				</label>
 				<d2l-input-number
 					id="points_input_${question.id}"
-					class="points_input_field"
 					label=${this.localize('inputLabelPoints')}
 					value=${ question.points }
 					@change=${this._validation}
-					min = 1
+					min = 0
```

Comment:
min=0? 
Also, did they remove the ability to suggest changes??

Reply:
it is minimum 0 but excluding 0

Reply:
I meant the spacing.  Ooh, it's back.
```suggestion
					min=0
```